src/MySqlConnector/MySqlClient/MySqlConnectionStringBuilder.cs

```diff
@@ -118,6 +118,12 @@ public bool UseAffectedRows
 			set { MySqlConnectionStringOption.UseAffectedRows.SetValue(this, value); }
 		}
 
+		public bool Synchronous
```

Comment:
`ForceSynchronous` might convey the meaning of this setting better; the default (`Synchronous = false`) doesn't stop the library being "synchronous".